<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="icons.qrc">:/sound64.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="soundButton">
              <property name="text">
               <string>Sounds</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="soundLabel">
              <property name="toolTip">
               <string>number of files.max 501</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="icons.qrc">:/pic64.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="picButton">
              <property name="text">
               <string>Pictures</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="picLabel">
              <property name="toolTip">
               <string>number of files.max 501</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="icons.qrc">:/video64.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="videoButton">
              <property name="text">
               <string>Videos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="videoLabel">
              <property name="toolTip">
               <string>number of files.max 501</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="icons.qrc">:/docs64.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="docButton">
              <property name="text">
               <string>docs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="docLabel">
              <property name="toolTip">
               <string>number of files.max 501</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="text">
           <string>open another directory</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="view">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Pictures View</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
